client/src/components/EventPage/Comments.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'react-materialize';
+import { Link } from 'react-router-dom';
+import moment from 'moment';
+import CommentForm from './CommentForm';
+import { request } from '../../utils';
+
+class Comments extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: [],
+    };
+    this.addComment = this.addComment.bind(this);
+  }
+
+  componentDidMount = () => this.getComments();
+
+  getComments() {
+    request(`/comment/${this.props.eventId}`).then(data => {
+      if ('comments' in data && data.comments.length) {
+        this.setState({ comments: data.comments });
+      }
+    })
+  }
+
+  showAnswerForm = e => e.currentTarget.parentNode.parentNode.lastChild.style.display = 'block';
+
+  addComment = e => {
+    let forms = document.getElementsByClassName('answer-form');
+    for (let i = 0; i < forms.length; i++) {
+      forms[i].style.display = 'none';
+    }
+
+    const parentCommentId = e.currentTarget.getAttribute('data-id');
+    let comment = {
+      comment: document.getElementById(`textarea-${parentCommentId}`).value,
+      unix_time: Date.now(),
```

Comment:
It's a very bad idea to add timestamps on client side. Never trust data coming from outside, add such info on backend side.